mobilizon_bots/storage/query.py

```diff
@@ -17,22 +18,30 @@
 CONNECTION_NAME = "models" if "pytest" in sys.modules else None
 
 
+async def load_events(queryset: QuerySet[Event]) -> list[Event]:
```

Comment:
add _ to mark it as internal. The name might be misleading otherwise (or find a more clear name)